client/src/partials/cash/modals/transfer.modal.html

```diff
@@ -70,13 +70,11 @@
 
     <button
       type="button"
-      class="btn btn-default"
+      class="pull-left btn btn-default"
       ng-click="CashTransferModalCtrl.cancel()"
       data-transfer-modal-cancel>
       {{ "FORM.BUTTONS.CANCEL" | translate }}
     </button>
+    <div class="clearfix"></div>
```

Comment:
This "clearfix" class can go on the parent element. So, if you have this code:
```html
<div class="modal-footer">
  <button class="btn btn-default pull-right"></button>
</div>
```
And the button will not stay in the modal footer, this is a better way to fix it:
```html
<!-- put the clearfix on the parent -->
<div class="modal-footer clearfix">
  <button class="btn btn-default pull-right"></button>
  <!-- <div class="clearfix"></div> would also work, but is less clear, I think -->
</div>
```